config/crds/sources_v1alpha1_containersource.yaml

```diff
@@ -38,6 +38,8 @@ spec:
             image:
               minLength: 1
               type: string
+            serviceAccountName:
+              type: string
```

Comment:
These serviceAccountName changes look unrelated to this specific fix - probably slipped in when generating the manifests based on some local changes to the container and kubernetes event source types?

Reply:
someone forgot to run the gen command.

Reply:
@grantr ^^

but I am not sure why the verify script did not catch this.

Reply:
:hand: This should have gone into #60, but I'm also curious why the verify script didn't work.